sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Changed return type of `send_message` to `SendChatMessageResult`.
 - Replaced `CommunicationError` with `ChatError`.
 - Refactored `CommunicationTokenCredential` constructor to accept `token` instead of `CommunicationTokenRefreshOptions`.
+- Renamed `ChatThreadParticipant` to `ChatParticipant`.
```

Comment:
Should we add the other renames for the properties and arguments too?
And mention that communication identifiers now have been redesigned to use a dictionary property bag?

Reply:
Updated the Changelog to reflect the recent changes made